### DISCUSSION TITLE
Extract Greek letters from a PDF Document

### DISCUSSION THREAD

**Discussion by mohitrobo** [pymupdf/PyMuPDF#1925] [Looking for help]
Text extraction from a PDF file:-
1.) Used the syntax **Page.get_text()** with the clip to define the rectangle to extract the text.
2.) The PDF text had some mathematical equations with greek letters.
3.) The returned text returns text with **?** in place of the greek letters.

How to resolve the issue?

**Comment by JorjMcKie** [comment]
Not possible unfortunately. The font used for the letters is accompanied with no backtranslation info (usually stored in `/ToUnicode` - the CMAP table).
All you could do is OCR-ing those text pieces. How such "partial" OCR-ing works is explained [here](https://github.com/pymupdf/PyMuPDF-Utilities/blob/master/OCR/tesseract2.py).
